packages/dd-trace/src/appsec/rule_manager.js

```diff
@@ -14,6 +15,60 @@ function applyRules (rules, config) {
   appliedCallbacks.set(rules, callback)
 }
 
+function updateAsmData (action, asmData, asmDataId) {
+  if (action === 'unapply') {
+    appliedAsmData.delete(asmDataId)
+  } else {
+    appliedAsmData.set(asmDataId, asmData)
+  }
+
+  const mergedRuleData = mergeRuleData(appliedAsmData.values())
+  for (const callback of appliedCallbacks.values()) {
+    callback.updateRuleData(mergedRuleData)
+  }
+}
+
+function mergeRuleData (asmDataValues) {
+  const mergedRulesData = new Map()
+  for (const asmData of asmDataValues) {
+    if (!asmData.rules_data) continue
```

Comment:
Curly brackets missing?

Reply:
I think that in that inlined `continue` it is ok at this way